main.go

```diff
@@ -285,21 +285,20 @@ func main() {
 		return
 	}
 
-	originPath := flag.Arg(0)
+	originPaths := flag.Args()
 
 	if filePath != "" {
-		deprecatedMessagesCh <- fmt.Sprintf("-%s is deprecated. Put file name as last argument to the command(Example: goimports-reviser -rm-unused -set-alias -format goimports-reviser/main.go)", filePathArg)
-		originPath = filePath
+		deprecatedMessagesCh <- fmt.Sprintf("-%s is deprecated. Put file name(s) as last argument to the command(Example: goimports-reviser -rm-unused -set-alias -format goimports-reviser/main.go)", filePathArg)
+		originPaths = append(originPaths, filePath)
 	}
 
-	if originPath == "" {
-		originPath = reviser.StandardInput
-	}
-
-	if err := validateRequiredParam(originPath); err != nil {
-		fmt.Printf("%s\n\n", err)
-		printUsage()
-		os.Exit(1)
+	if len(originPaths) == 0 || (len(originPaths) == 1 && originPaths[0] == "-") {
+		originPaths[0] = reviser.StandardInput
+		if err := validateRequiredParam(originPaths[0]); err != nil {
+			log.Printf("%s\n\n", err)
+			printUsage()
+			os.Exit(1)
+		}
 	}
```

Comment:
Shouldn't be this?


```suggestion
 	}
	if err := validateRequiredParam(originPaths[0]); err != nil {
		log.Printf("%s\n\n", err)
		printUsage()
		os.Exit(1)
	}
```

Reply:
I think I'm possibly misunderstanding something here. `validateRequiredParam()` will always return nil if the passed argument is anything other than `reviser.StandardInput`:

```go
func validateRequiredParam(filePath string) error {
	if filePath == reviser.StandardInput {
		stat, _ := os.Stdin.Stat()
		if stat.Mode()&os.ModeNamedPipe == 0 {
			// no data on stdin
			return errors.New("no data on stdin")
		}
	}
	return nil
}
```

So I think there's no point in calling it unless we know that we're operating on stdin, which would only happen if we find ourselves on the inside of `if len(originPaths) == 0 || (len(originPaths) == 1 && originPaths[0] == "-") {`